### PR TITLE
OBSProject.OBSStudio version 27.2.1

### PR DESCRIPTION
**Pull request by krystofkrticka**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/47583)

### CODE REVIEW

File: manifests/o/OBSProject/OBSStudio/27.2.1/OBSProject.OBSStudio.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  • Fixed a performance issue with X11 window capture on Linux [ishitatsuyuki]
+
+  • Fixed a bug where projectors could not be resized with certain window managers on Linux [univrsal]
+
+  • Fixed window flashing on macOS when starting the program minimized to tray [CodyJung]
+
+  • Fixed a bug where push-to-talk or mute delay would not save under certain circumstances [CodyJung]
+
+  • Fixed a bug where mouse hotkeys would not always function correctly on Linux [kkartaltepe]
+ReleaseNotesUrl: https://github.com/obsproject/obs-studio/releases/tag/27.2.0
```
Comment (OfficialEsco) [inline manifests/o/OBSProject/OBSStudio/27.2.1/OBSProject.OBSStudio.locale.en-US.yaml:163]: Hey, just for later, please check if these fields are defined

File: manifests/o/OBSProject/OBSStudio/27.2.1/OBSProject.OBSStudio.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/obsproject/obs-studio/releases/download/27.2.1/OBS-Studio-27.2.1-Full-Installer-x64.exe
+  InstallerSha256: E1277A2A7BA6252D2029667D70BC2E79CC9ED15E2823E6C1FFBF355DEBE98D63
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/obsproject/obs-studio/releases/download/27.2.1/OBS-Studio-27.2.1-Full-Installer-x86.exe
+  InstallerSha256: 32B33B75F3F7E69594ABDB4C01868ADCDE12C52BB9C30A6566AE165E85EE56D0
+ManifestType: installer
+ManifestVersion: 1.1.0
+ReleaseDate: 2022-02-14
```
Comment (OfficialEsco) [inline manifests/o/OBSProject/OBSStudio/27.2.1/OBSProject.OBSStudio.installer.yaml:28]: and the ReleaseDate field

File: manifests/o/OBSProject/OBSStudio/27.2.1/OBSProject.OBSStudio.locale.en-US.yaml (hunk 1 above)
Comment (krystofkrticka) [reply manifests/o/OBSProject/OBSStudio/27.2.1/OBSProject.OBSStudio.locale.en-US.yaml]: Done, manually updated those, release notes.

File: manifests/o/OBSProject/OBSStudio/27.2.1/OBSProject.OBSStudio.installer.yaml (hunk 2 above)
Comment (krystofkrticka) [reply manifests/o/OBSProject/OBSStudio/27.2.1/OBSProject.OBSStudio.installer.yaml]: Done already corrected to same as release notes.

**Review by OfficialEsco — DISMISSED** [review]
🚀

**Comment by OfficialEsco** [comment]
Hmm, GitHub preview looks normal but the raw looks weird https://raw.githubusercontent.com/microsoft/winget-pkgs/4498fbe9873346e27643e9e3637a1c3de2c86532/manifests/o/OBSProject/OBSStudio/27.2.1/OBSProject.OBSStudio.locale.en-US.yaml

**Comment by OfficialEsco** [comment]
AHH It added Tabs instead of spaces
Code_kG2oK60UkD

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**